state/state_test.go

```diff
@@ -77,6 +77,45 @@ func TestCacheAndUnpackHooksRunOnMutablePackage(t *testing.T) {
 	assert.Equal(t, os.FileMode(0500), info.Mode()&0777, info.Mode().String())
 }
 
+func TestUnpacksMissingBinaries(t *testing.T) {
```

Comment:
Can you add an integration test to `integration/` too (or instead) please?

Reply:
Can do!

Reply:
How do I change a package definition or introspect state in the integration tests?